ff4j-spring-boot-web-api/pom.xml

```diff
@@ -67,26 +67,27 @@
     <dependency>
       <groupId>org.assertj</groupId>
       <artifactId>assertj-core</artifactId>
+      <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.skyscreamer</groupId>
       <artifactId>jsonassert</artifactId>
+      <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>info.cukes</groupId>
       <artifactId>cucumber-junit</artifactId>
+      <scope>test</scope>
```

Comment:
This is a[lready in test scope](https://github.com/ff4j/ff4j-spring-boot-starter-parent/blob/master/pom.xml#L211)